Makefile

```diff
@@ -237,16 +242,47 @@ test-coverage: setup-envtest; $(info  running coverage tests...) @ ## Run covera
 # Container image
 .PHONY: image
 image: ; $(info Building Docker image...)  @ ## Build container image
-	$(IMAGE_BUILDER) build --build-arg BUILD_DATE="$(BUILD_TIMESTAMP)" \
+	$Q DOCKER_BUILDKIT=1 $(IMAGE_BUILDER) build --build-arg BUILD_DATE="$(BUILD_TIMESTAMP)" \
 		--build-arg VERSION="$(BUILD_VERSION)" \
 		--build-arg VCS_REF="$(VCS_REF)" \
 		--build-arg VCS_BRANCH="$(VCS_BRANCH)" \
 		--build-arg LDFLAGS=$(LDFLAGS) \
+		--build-arg ARCH="$(ARCH)" \
+		--build-arg GCFLAGS="$(GCFLAGS)" \
 		-t $(TAG) -f $(DOCKERFILE)  $(CURDIR) $(IMAGE_BUILD_OPTS)
 
 image-push:
 	$(IMAGE_BUILDER) push $(TAG)
 
+# Container image
+.PHONY: image-build
+image-build: ; $(info Building Docker image...)  @ ## Build container image
+	DOCKER_BUILDKIT=1 $(IMAGE_BUILDER) build --build-arg BUILD_DATE="$(BUILD_TIMESTAMP)" \
+		--build-arg VERSION="$(BUILD_VERSION)" \
+		--build-arg VCS_REF="$(VCS_REF)" \
+		--build-arg VCS_BRANCH="$(VCS_BRANCH)" \
+		--build-arg LDFLAGS=$(LDFLAGS) \
+		--build-arg ARCH="$(ARCH)" \
+		--build-arg GCFLAGS="$(GCFLAGS)" \
+		-t $(CONTROLLER_IMAGE)-$(ARCH):$(VERSION) -f $(DOCKERFILE)  $(CURDIR) $(IMAGE_BUILD_OPTS)
+
+image-build-%:
+	make ARCH=$* image-build
+
+.PHONY: image-build-multiarch
+image-build-multiarch: $(addprefix image-build-,$(BUILD_ARCH))
```

Comment:
this target builds two images `network-operator-amd64:master` and `network-operator-arm64:master` instead of one multi-arch image like `docker buildx` does.  That's not desirable pattern because we can't deliver single helm chart and OCP bundle for different platforms

Reply:
It's the target `image-push-multiarch` that creates a single tag for multiarch platforms. `image-build-multiarch` + `image-push-multiarch` should produce something identical to what `buildx` produces. We can also put both into the same target, or at least put the `manifest create` part into this target so that it's created locally.